layouts/data-playground/single.html

```diff
@@ -2,10 +2,144 @@
 {{ .Title }} &ndash; {{ .Site.Title }}
 {{ end }}
 
+{{ define "header" }}
+<div class="relative bg-[#1D65A6]">
+    {{ partial "site-nav.html" . }}
+</div>
+{{ end }}
+
 {{ define "main" }}
 
 <div class="max-w-7xl mx-auto px-4 relative">
 
-</div>
+    <div class="w-full bg-[#F5F5F5] flex justify-between my-5 place-items-center h-36 px-10">
+
+        <div class="flex flex-col font-bold ">
+            <h2 class="text-xl">{{.Params.title}}</h2>
+            <h3 class="text-lg">{{.Params.subtitle}}</h3>
+        </div>
+        <div>
+            <a href="{{.Params.github_link}}" target="_blank"
+                class="bg-black font-bold  px-3 sm:px-8 py-4 rounded-md text-sm text-white  md:text-base flex place-content-center gap-4 hover:bg-neutral-900">
+                <span class="text-sm sm:text-base">
+                    Get Started
+                </span>
+                <span class="hidden sm:flex">
+                    <img src="img/data-playground/github.png" alt="github icon" height="24" width="24">
+                </span>
+            </a>
+
+        </div>
+
+    </div>
+
+    <div class="w-full flex justify-between px-10">
+
+        <div class="w-1/2 ">
+            <div class="flex flex-col my-4">
+                <p class="text-xl font-bold mb-5">About This Data Source</p>
+                <p class="text-base">
+
+                    {{ .Params.description}}
+                </p>
+            </div>
+            <div class="flex flex-col my-4">
+                <p class="text-lg font-bold">Producer</p>
+                <p class="text-base">
+                    <a class="cursor-pointer underline" href="{{.Params.producer_website}}">
+                        {{ .Params.producer_name}}
+                </p>
+                </a>
+            </div>
+            <div class="flex flex-col my-4">
+                <p class="text-lg font-bold">Data</p>
+                <p class="text-base">{{ .Params.data }}</p>
+            </div>
+            <div class="flex flex-col my-4">
+                <p class="text-lg font-bold">Account Required</p>
+                <p class="text-base">
+                    {{ if eq .Params.account_required "true"
+                    }}
+                    Yes
+                    {{
+                    else
+                    }}
+                    No
+                    {{
+                    end
+                    }}
+                </p>
+            </div>
+            <div class="flex flex-col my-4">
+                <div class="flex flex-row gap-2 ">
+                    <p class="text-lg font-bold">License</p>
+                    <button data-attr="tooltip" id="tooltip" class="h-[24px] w-[24px]">
+                        <img src="img/data-playground/tooltip.svg" alt="tooltip" width="24" height="24">
```

Comment:
I wasn't running the server before I looked at the changes, so I don't understand how stopping and re-starting it would solve the issue. However, I did that and I see script, but I don't see the warning text when I hover over the icon on the dc-metro page. Since the warning was added to that file above, it should work on that page.

I didn't see the hover action tested in the video that was included, so if you could provide one showing that it works that would be helpful.

![Screenshot 2023-06-09 at 11 30 59 AM (2)](https://github.com/rotationalio/rotational.io/assets/94616884/621802d8-d416-455a-aa4a-dff2c01d71b8)

https://www.awesomescreenshot.com/video/18192322?key=7f71c3ef4ce7a5f6c5613a7ca77de75a

Reply:
> I wasn't running the server before I looked at the changes, so I don't understand how stopping and re-starting it would solve the issue. However, I did that and I see the script, but I don't see the warning text when I hover over the icon on the dc-metro page. Since the warning was added to that file above, it should work on that page.
> 
 @daniellemaxwell I kindly request you to restart the server in order to ensure that the script is updated. We have observed that after making changes to the JavaScript (js) file, the updated version is not always reflected immediately during runtime. As you may recall, we encountered a similar issue in the past with the "from" feature, and had to refresh multiple times to see the updated js.
 
 
> I didn't see the hover action tested in the video that was included, so if you could provide one showing that it works that would be helpful.
> 
> ![Screenshot 2023-06-09 at 11 30 59 AM (2)](https://user-images.githubusercontent.com/94616884/244738945-621802d8-d416-455a-aa4a-dff2c01d71b8.png)
> 
> https://www.awesomescreenshot.com/video/18192322?key=7f71c3ef4ce7a5f6c5613a7ca77de75a